src/Stratis.Bitcoin.Features.RPC/Controllers/FullNodeController.cs

```diff
@@ -76,6 +83,14 @@ public Task Stop()
             return Task.CompletedTask;
         }
 
+        /// <summary>
+        /// Retrieves a transaction given a transaction hash in either simple or verbose form.
+        /// </summary>
+        /// <param name="txid">The transaction hash.</param>
+        /// <param name="verbose">Non-zero if verbose model wanted.</param>
+        /// <returns>A <see cref="TransactionBriefModel"/> or <see cref="TransactionVerboseModel"/> as specified by verbose. <c>null</c> if no transaction matching the hash.</returns>
+        /// <exception cref="ArgumentException">Thrown if txid is invalid uint256.</exception>"
+        /// <remarks>Requires txindex=1, otherwise will return <c>null</c>.</remarks>
```

Comment:
without txindex=1 only txes that spend or create UTXOs for a wallet are indexed. So getrawtx may be used on such txes and it wont return null

Reply:
Gotcha. Mainly added this remark as the question pops up on discord about once/week. Have updated remark with this clarification and also transferred it over to the API method comments.